### PR TITLE
Bump vscode

### LINKED ISSUE #79: Updates to support VS Code and Extensions
The docker session image should be modified as followed:

1. Change from installing the session from the random tar file to actually using the RSP Debian. This will include the Rstudio-server binary to help install VS code and extensions

2. Remove the lines to manually install code server, and instead invoke the appropriate RSP install command.
    2a. Caveat one: decide if we should use global extension installs (and install the Python and RStudio extensions) OR opt into user extension land and just install vs code and indicate we want the Rstudio extension installed at session start. I'm leaning towards the global approach
    2b. Caveat two: we should install python-dotenv since thats required for the Rstudio extension 
    2c. Caveat three: we need to figure out if vscode.conf can be eliminated from the session image
   

Sample Dockerfile for bionic attached.

May want to wait for https://github.com/rstudio/rstudio-pro/issues/2564 or https://github.com/rstudio/rstudio-pro/issues/2563

[Dockerfile.txt](https://github.com/rstudio/docker-r-session-complete/files/6430156/Dockerfile.txt)

### PR DESCRIPTION
**Pull request by colearendt**
Switch to paradigm highlighted in #79 

Close #79

### CODE REVIEW

**Comment by MariaSemple** [comment]
I tested this locally and was able to build both the bionic and centos7 images successfully.

File: Makefile [hunk 1]
```diff
@@ -15,7 +15,3 @@ update-versions:  ## Update the version files for all products
 	@sed -i '' "s/^RSP_VERSION=.*/RSP_VERSION=${RSP_VERSION}/g" r-session-complete/centos7/.env
 	@sed -i '' "s/^ARG RSP_VERSION=.*/ARG RSP_VERSION=${RSP_VERSION}/g" r-session-complete/bionic/Dockerfile
 	@sed -i '' "s/^ARG RSP_VERSION=.*/ARG RSP_VERSION=${RSP_VERSION}/g" r-session-complete/centos7/Dockerfile
-	@sed -i '' "s/^CODE_SERVER_VERSION=.*/CODE_SERVER_VERSION=${CODE_SERVER_VERSION}/g" r-session-complete/bionic/.env
-	@sed -i '' "s/^CODE_SERVER_VERSION=.*/CODE_SERVER_VERSION=${CODE_SERVER_VERSION}/g" r-session-complete/centos7/.env
-	@sed -i '' "s/^ARG CODE_SERVER_VERSION=.*/ARG CODE_SERVER_VERSION=${CODE_SERVER_VERSION}/g" r-session-complete/bionic/Dockerfile
-	@sed -i '' "s/^ARG CODE_SERVER_VERSION=.*/ARG CODE_SERVER_VERSION=${CODE_SERVER_VERSION}/g" r-session-complete/centos7/Dockerfile
```
Comment (MariaSemple) [inline Makefile:21]: Should an issue be opened to repair these tests for the future?

File: r-session-complete/bionic/Dockerfile [hunk 2]
```diff
@@ -27,14 +24,18 @@ RUN apt-get update -y && \
...
-    echo "${RSP_VERSION}" > /usr/lib/rstudio-server/SESSION_VERSION
+ARG RSP_VERSION=1.4.1106-5
+ARG RSP_NAME=rstudio-server-pro
+ARG RSP_DOWNLOAD_URL=https://s3.amazonaws.com/rstudio-ide-build/server/bionic/amd64
+RUN apt-get update --fix-missing \
+    && apt-get install -y gdebi-core \
+    && curl -O ${RSP_DOWNLOAD_URL}/${RSP_NAME}-${RSP_VERSION}-amd64.deb \
+    && gdebi --non-interactive ${RSP_NAME}-${RSP_VERSION}-amd64.deb \
+    && rm ${RSP_NAME}-${RSP_VERSION}-amd64.deb \
+    # && apt-get remove gdebi-core -y \
```
Comment (MariaSemple) [inline r-session-complete/bionic/Dockerfile]:
```suggestion
```

File: r-session-complete/bionic/test/goss.yaml [hunk 3]
```diff
@@ -5,18 +5,12 @@ file:
...
-    exit-status: 0
-    stdout: [
-      "/{{ .Env.RSP_VERSION }}/",
-    ]
+  # "cat /usr/lib/rstudio-server/SESSION_VERSION":
+  #   title: session_version_matches
+  #   exit-status: 0
+  #   stdout: [
+  #     "/{{ .Env.RSP_VERSION }}/",
+  #   ]
```
Comment (MariaSemple) [inline r-session-complete/bionic/test/goss.yaml]:
```suggestion
```

File: r-session-complete/centos7/test/goss.yaml [hunk 4]
```diff
@@ -5,18 +5,12 @@ file:
...
-    exit-status: 0
-    stdout: [
-      "/{{ .Env.RSP_VERSION }}/",
-    ]
+  # "cat /usr/lib/rstudio-server/SESSION_VERSION":
+  #   title: session_version_matches
+  #   exit-status: 0
+  #   stdout: [
+  #     "/{{ .Env.RSP_VERSION }}/",
+  #   ]
```
Comment (MariaSemple) [inline r-session-complete/centos7/test/goss.yaml]:
```suggestion
```

**Review by MariaSemple — APPROVED** [review]
LGTM! Just a couple nits

File: Makefile (hunk 1 above)
Comment (colearendt) [reply Makefile:21]: These aren't tests, but yeah, this ultimately requires a mapping of "what version of vscode works with what version of RStudio" if we want to test the version

**Pull request merged**